tendenci/apps/events/models.py

```diff
@@ -881,16 +881,27 @@ def can_edit_child_events(self):
         if not self.event.nested_events_enabled:
             return False
 
+        can_edit = False
         for registrant in self.registrant_set.filter(cancel_dt__isnull=True):
-            return (
+            can_edit = (
                 not registrant.registration_closed and (
                     (registrant.user and registrant.user.profile.is_superuser and
                     registrant.event.child_events.exists()) or
                     registrant.event.upcoming_child_events.exists()
                 )
             )
+            if can_edit:
+                return True
```

Comment:
Fixed a bug that I saw here... previously it would return False if the first registrant it hit didn't meet the criteria even though the 2nd one might. This turns on the button to Save and Edit Sessions even without changing attendance dates